server/src/models/checkInModel.ts

```diff
@@ -0,0 +1,25 @@
+import { Schema, model } from "mongoose";
+
+const checkInSchema = new Schema({
+  _id: {
+    type: Schema.Types.ObjectId,
+    required: true,
+  },
+  isChecked: {
+    type: Boolean,
+    default: false,
+    required: true,
+  },
+  remindIn: {
+    type: Number,
+    default: 120,
+    required: true,
+  },
+  lastUpdatedAt: {
```

Comment:
i don't think you need a remindin and a lastUpdated field. Why do you need both?